src/oscar/static_src/oscar/scss/page/checkout.scss

```diff
@@ -125,11 +93,8 @@ input[type="text"][id*="quantity"], input[type="number"][id*="quantity"] {
 
 .shipping-payment {
   .card {
-    margin-bottom: $base-margin-bottom;
```

Comment:
I think this needs to remain?

Reply:
Yeah, you're right, sorry. I've added it back.